src/Slider.js

```diff
@@ -5,18 +5,14 @@ import mdlUpgrade from './utils/mdlUpgrade';
 class Slider extends React.Component {
     static propTypes = {
         className: PropTypes.string,
-        max: PropTypes.number.isRequired,
-        min: PropTypes.number.isRequired,
-        onChange: PropTypes.func.isRequired,
+        max: PropTypes.number,
```

Comment:
But it's allowed. None of those are required properties on input elements. For example, `<input type="range" max="100" />` is valid.

Also, consider `<Slider disabled />`